src/app/child-dev-project/children/children-list/list-attendance/list-attendance.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, Input } from "@angular/core";
+import { AttendanceMonth } from "../../../attendance/model/attendance-month";
+import { ChildrenService } from "../../children.service";
+import { Child } from "../../model/child";
+import { MediaChange, MediaObserver } from "@angular/flex-layout";
+import { OnInitDynamicComponent } from "../../../../core/view/dynamic-components/on-init-dynamic-component.interface";
+
+@Component({
+  selector: "app-list-attendance",
+  template: `
+    <app-attendance-block
+      *ngFor="
+        let att of attendanceList
+          | filterBy: { institution: filterBy }
+          | slice: 0:maxAttendanceBlocks
+      "
+      [attendanceData]="att"
+    ></app-attendance-block>
+  `,
+})
+export class ListAttendanceComponent implements OnInitDynamicComponent {
```

Comment:
We could do that but on the other hand it would limit the components in their usage. The `ChildBlock` component is used in many locations, not just as a column.

Reply:
Oh, I misinterpreted the use of this a bit, maybe. As I understand it now this component (loads and) displays the most recent attendance blocks for one child, right?

Then maybe naming it something like `RecentAttendanceBlocksComponent` or `RecentAttendanceBlocksOfChildComponent`?

This should also go into a jsdoc for the component.